templates/scraper.py

```diff
@@ -1,31 +1,35 @@
+from typing import List, Optional
+
 from ._abstract import AbstractScraper
 
 
 class Template(AbstractScraper):
+    # Functions should return `None` if they cannot access
+    # the correct value as oppposed to `""` or `-1` etc.
```

Comment:
I'm happy to express this information in any way, I thought this would be the most visible to people who just want to create a scraper as fast as possible.